scio-core/src/main/scala/com/spotify/scio/values/PairSCollectionFunctions.scala

```diff
@@ -624,6 +626,30 @@ class PairSCollectionFunctions[K, V](val self: SCollection[(K, V)])
     new MapSideInput[K, V](o)
   }
 
+  def asSparkeySideInput: SideInput[Map[String, String]] = {
```

Comment:
I thought about this and decided not to because the users would have to explicitly pull in the Sparkey dependency - but I guess that's no such a big deal?